### DISCUSSION TITLE
Add option to create automatically connection shortcut on Desktop in Linux, too ...

### DISCUSSION THREAD

**Discussion by givbra** [rustdesk/rustdesk#5772] [Feature Request]
Add option to create automatically connection shortcut on Desktop in Linux, too, like it is in the Windows version ...
Now, on Linux, it is possible only if we create shortcut to RustDesk manually with command "rustdesk --connect ID" ...

![2022-06-22_03-29](https://user-images.githubusercontent.com/107966747/174918206-31966f7d-c549-4ad4-9830-6e2ddd013230.png)

**Comment by halacs** [comment]
Note for those who are looking for a workaround like me until this feature gets implemented on Linux too.

On Linux, RustDesk has `--connect` CLI argument (don't know where it is documented, if any) can be used to create shortcut manually.

Example:

`rustdesk --connect [PASTE-YOUR-ID-HERE] --password [PASTE-YOUR-PASSWORD-HERE-IF-NOT-YET-SAVED]`

Source: https://github.com/rustdesk/rustdesk/discussions/7049
